classroom.rb

```diff
@@ -0,0 +1,14 @@
+class ClassRoom
+  att_accessor :label
+  att_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md) as you have created the `has-many` side (a classroom has many students). kindly create the `belongs-to` side (a student belongs to a classroom) also in `student.rb`. 🤗

```ruby
def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
end
```

So that any time we add a student to a classroom the classroom for the student is set as well, and vice versa. (i.e both ways relationships)